README.md

```diff
@@ -41,9 +43,9 @@ await client.sendData(IlpPacket.serializeIlpPrepare({
 Two functions must be defined in order for the plugin to handle money.
 
 * `sendMoney (amount: string) -> Promise<null>`: sends `amount` of units to the peer. This should be done via a BTP `TRANSFER` call.
-* `handleMoney (from: null, { requestId: number, data: { amount: string, protocolData: Array<ProtocolData> } }) -> Array<ProtocolData>`: This function is called on an incoming BTP `TRANSFER`.
+* `_handleMoney (from: string, btpPacket: BtpPacket) -> Promise<Array<BtpSubProtocol>>`: This function is called on an incoming BTP `TRANSFER`.
 
-ProtocolData is made up of:
+BtpSubProtocol is made up of:
```

Comment:
Why the name change? I think we had that discussion already in the initial BTP RFCs PR

Reply:
It's called `ProtocolData` in https://github.com/interledger/rfcs/blob/master/asn1/BilateralTransferProtocol.asn#L38 so best use that same name in the implementation.

Reply:
This is a TypeScript interface name. It allows someone reading the README to easily find the TypeScript interface and I think it's bad practice to name classes/interfaces too generically otherwise they are likely to conflict with others.